Winapp2.ini

```diff
@@ -3746,9 +3755,9 @@ FileKey4=%SystemDrive%|wifi.log
 
 [Atheros Driver *]
 LangSecRef=3021
-Detect=HKLM\Atheros
+Detect=HKLM\SOFTWARE\Atheros
 Default=False
-FileKey1=%CommonAppData%\Atheros|*.log|RECURSE
+FileKey1=%CommonAppData%\Atheros|*.log;InstallLog.txt|RECURSE
 FileKey2=%CommonAppData%\Qualcomm Atheros WiFi Driver Installation|*.Log|RECURSE

```

Comment:
In the previous pull request it was "%ProgramFiles%\\Common Files\\Atheros|InstallLog.txt" = "%CommonProgramFiles%\\Atheros|InstallLog.txt".

What is correct? %CommonAppData% or %CommonProgramFiles%?